From 992af43037aa9c85109576d8b7581a8df1c200de Mon Sep 17 00:00:00 2001
From: Shawn Xiao <b49994@freescale.com>
Date: Mon, 2 Feb 2015 15:39:51 +0800
Subject: [PATCH 1091/1691] MGS-1276 ENGR00306443-2 dts:Add QoS description in
 imx6sx dts

commit 992af43037aa9c85109576d8b7581a8df1c200de from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

(cherry-pick from f9759787e60ad3422d2119f9f25ac320ca58f5df)

confilict: arch/arm/boot/dts/imx6sx.dtsi
The dts file arch has changed in 3.14. Add QoS description in
imx6sx dts manually and solve the conflict.

Date Feb 2, 2015

Signed-off-by: Shawn Xiao <b49994@freescale.com>
---
 arch/arm/boot/dts/imx6sx.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 1a2a217..4b7e511 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -1208,6 +1208,11 @@
 				clocks = <&clks IMX6SX_CLK_I2C4>;
 				status = "disabled";
 			};
+
+			qosc: qosc@021fc000 {
+				compatible = "fsl,imx6sx-qosc";
+				reg = <0x021fc000 0x4000>;
+			};
 		};
 
 		aips3: aips-bus@02200000 {
-- 
1.9.1

